packages/images/lib/optimize.js

```diff
@@ -36,32 +35,105 @@ function getDirectoriesRecursive(srcPath) {
   ];
 }
 
+/**
+ * @param {Function} resolve
+ * @param {Function} reject
+ * @param {string} imagePath
+ * @param {string} [type]
+ */
+async function handleImage(resolve, reject, imagePath, type) {
+  try {
+    const image = await imagePool.ingestImage(imagePath);
+    const options = {};
+
+    if (type) {
+      options[type] = {};
```

Comment:
You could write that empty object to a const, like `const defaultSettings = {}`. I needed to check the API docs to get why it needs to be an empty object.